src/api/featureApi.js

```diff
@@ -15,6 +15,13 @@ const featureApi = {
     });
   },
 
+  async getSignalBits(block) {
+    return requestExplorerServiceV1.get(`node_api/feature`, { params: { block } }).then(res => {
+      return res.data.signal_bits
+    });
```

Comment:
This is very similar to the getFeatures above.
Why not have a `getFeatures(block = null)` and have the getSignalBits method call it?

Or maybe use react state to call the api only once and have getSignalBits as a selector of the `signal_bits`.
What do you think?

Reply:
Sure! Done in https://github.com/HathorNetwork/hathor-explorer/pull/244/commits/c78ca0f1ff3118c0ac115e302d0f8dc15d8c651e